Applet/src/com/android/javacard/keymaster/KMKeymintDataStore.java

```diff
@@ -963,6 +977,9 @@ private void handleUpgrade(Element element, short oldVersion) {
     attIdProduct = (byte[]) element.readObject();
     attIdSerial = (byte[]) element.readObject();
     attIdImei = (byte[]) element.readObject();
+    if (oldVersion >= KM_APPLET_PACKAGE_VERSION_3) {
+        attIdSecondImei = (byte[]) element.readObject();
+    }
```

Comment:
Shouldn't the object count be increased by 1 as secondImei is added in getBackupObjectCount() ?